duckduckgo/iOS
# Update to privacy config v2

Task/Issue URL: https://app.asana.com/0/1200890834746050/1201683113648489/f
Tech Design URL:
CC: @bwaresiak @jonathanKingston @brindy 

**Description**:
This PR updates the privacy config url to use v2 and updates the embedded config as well.

**Steps to test this PR**:
1. Run browser and check behavior is normal
2. Edit the local config (and force the config manager to use the local config) to use a minSupportedVersion for a feature with a version beyond the current version (e.g. 7.66.1.2)
3. That feature should appear disabled



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
